src/main/java/org/myrobotlab/service/InMoov2Arm.java

```diff
@@ -89,7 +89,7 @@ public static DHRobotArm getDHRobotArm(String name, String side) {
     return arm;
   }
   
-  public void onMoveArm(HashMap<String, Double> map) {
+  public void onMove(Map<String, Double> map) {

```

Comment:
do we / should we be concerned with backwards compatibility here?  perhaps instead just add this new method and have the old method name call it.. mark the old method (onMoveArm as deprecated? )

Reply:
mkay